src/lib.rs

```diff
@@ -1250,7 +1250,7 @@ impl<T: Counter> Histogram<T> {
     /// of bins needed to cover that range.
     ///
     /// May fail if `high` is not at least twice the lowest discernible value.
-    fn cover(&mut self, high: i64) -> Result<usize, &'static str> {
+    fn cover(&mut self, high: u64) -> Result<usize, &'static str> {
         if high < 2 * self.lowestDiscernibleValue {
             return Err("highest trackable value cannot be < (2 * lowest discernible value)");
```

Comment:
This seems like programmer error, so I think this could probably be a `panic!` or `assert!`. Existing usage seems to suggest that this is the case. It's used in two places: new histograms, where this check cannot possibly fail as it's already validated, and `resize`, where the result is simply `unwrap()`'d.